Core/src/org/sleuthkit/autopsy/modules/embeddedfileextractor/ImageExtractor.java

```diff
@@ -211,6 +213,13 @@ void extractImage(AbstractFile abstractFile) {
         HWPFDocument doc = null;
         try {
             doc = new HWPFDocument(new ReadContentInputStream(af));
+        } catch (OldFileFormatException ex) {
```

Comment:
I think we should combine the two "catch" blocks into a single catch block and move the comments there. The same applies to all other changes in the pull request.